big-int/src/operators/multiply.cpp

```diff
@@ -27,10 +27,102 @@
 
 #include <istream>
 #include <ostream>
-
-#include "../bigint.hpp"
+#include <bigint.hpp>
+#include <algorithm>
 
 namespace libbig
 {
+    largeInt largeInt::operator*(const largeInt &next_number) {
```

Comment:
still two overloads are missing.
for `largeInt * int` and `largeInt * int64_t` (`int64_t` aka `long long`)